packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -101,7 +101,9 @@ const FlipbookControls = ({
     resizeObserver.current.observe(controlsContainer.current)
 
     return () => {
-      resizeObserver.current.unobserve(controlsContainer.current)
+      if (controlsContainer.current) {
+        resizeObserver.current.unobserve(controlsContainer.current)
+      }
     }
```

Comment:
There's a subtle bug here. This function unobserves the current value of `controlsContainer`. You want to unobserve the DOM node that was observed when the effect originally ran. 

Add something like this to the body of the effect:

```js
const containerElement = controlsContainer.current)
```

Then observe and unobserve `containerElement`.

Otherwise, `controlsContainer.current` can mutate and break the effect hook.

Reply:
This might be cleaner. 

```js
useEffect(function () {
 // …declare debouncedObserver here
 const containerElement = controlsContainer.current
 const resizeObserver = new window.ResizeObserver(debouncedObserver)
 resizeObserver.observe(containerElement)

return () => {
  resizeObserver?.disconnect()
}
})
```